projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.QuoteTemplates.php

```diff
@@ -198,7 +197,7 @@ public function save_data( $quoteTemplateID, $quoteTemplate ) {
                 if (isset($_POST['zbs_quotetemplate_content'])) {
 
                     #} Save content
-                    $quoteTemplate['content'] = zeroBSCRM_io_WPEditor_WPEditorToDB($_POST['zbs_quotetemplate_content']);                        
+					$quoteTemplate['content'] = wp_kses( wp_unslash( $_POST['zbs_quotetemplate_content'] ), $zbs->acceptable_html ); //phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase,WordPress.Security.NonceVerification.Missing -- to follow up with.
```

Comment:
Why `wp_unslash()` here? This results in us losing backslashes on save.

Reply:
Also a PHPCS suggestion in this case, but I didn't spot the implications. There are additional slashes being added here, but they are removed before output in the editor (so wp_unslash is duplicating the removal). I think the best thing is to ignore that sniff here.